hermite.go

```diff
@@ -0,0 +1,383 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package quad
+
+import (
+	"math"
+	"math/cmplx"
+
+	"github.com/gonum/floats"
+	"github.com/gonum/mathext/airy"
+)
+
+// Hermite integrates a function over the whole real line with negative
```

Comment:
Is there another Hermite in this context? It should also be GaussLegendre, GaussJacobi, etc. The Gauss seems redundant